packages/samples/headless-ssr/src/app/ssr/components/search-page.tsx

```diff
@@ -4,7 +4,7 @@ import {
   SearchSSRState,
   SearchCSRState,
   hydrateInitialState,
-} from '@/src/common/engine';
+} from '@/src/app/ssr/common/engine';
```

Comment:
instead of `ssr`, I think I would prefer something like `generic`/`agnostic`, since:
* This sample is already in the `headless-ssr` sample package, the name may be redundant.
* Technically, the `react` samples are also `ssr`, so a different name may help differentiate it.

Reply:
Was thinking the same, renamed to `generic`